src/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "mattermost-desktop",
   "productName": "Mattermost",
   "desktopName": "Mattermost.desktop",
-  "version": "4.6.0-develop",
+  "version": "4.9.0-develop",
```

Comment:
I think you upgraded the version by mistake (most likely your own manual testing :) )

it is also the reason why the test are failing to build

Reply:
this has to go back to its previous value